### DISCUSSION TITLE
Discord Server?

### DISCUSSION THREAD

**Discussion by DickenSerm** [glanceapp/glance#149] [General]
I think a Discord server for the project would be great. I see a lot of potential in the project and I think that exchanging information via Discord works better than via the discussion area on github.

**Comment by svilenmarkov** [comment]
I've been reluctant to make a Discord server since I don't know if I'm going to have the time to moderate and be active in it, though I suppose it doesn't hurt to try.

https://discord.com/invite/7KQ7Xa9kJd

**Comment by DickenSerm** [comment]
Thanks, it was more about a relaxed exchange between users :)

**Comment by vishalkadam47** [comment]
@svilenmarkov create a Github webhook that will post updates from Github account to a channel in Discord. this would be very helpful to get notifications of new release
